build/docker/kitchen_terraform/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG BASE_IMAGE
 FROM $BASE_IMAGE
```

Comment:
This image file is actually obsolete; it should be removed. We use a shared image which is published to the Google Container Registry.

Reply:
About the first comment, I got it, so I'll fix it tomorrow morning.  About Dockerfile, which image I should use for it?

Reply:
Done.

Reply:
I think it makes sense to remove the container assembling, cause we use the standard container